src/test/java/com/auth0/client/mgmt/GuardianEntityTest.java

```diff
@@ -250,8 +251,12 @@ public void shouldUpdateGuardianSnsFactorProvider() throws Exception {
         assertThat(recordedRequest, hasHeader("Authorization", "Bearer apiToken"));
 
         Map<String, Object> body = bodyFromRequest(recordedRequest);
-        assertThat(body.size(), is(1));
-        assertThat(body, hasEntry("aws_region", (Object) "region"));
+        assertThat(body.size(), is(5));
+        assertThat(body, hasEntry("aws_access_key_id", "awsAccessKeyId"));
```

Comment:
yes because the body is `Map<String, Object>` and the matcher expects an object for the value.